package.json

```diff
@@ -79,6 +79,7 @@
   },
   "dependencies": {
     "@azure/cosmos-sign": "1.0.2",
+    "atob": "2.1.2",
```

Comment:
atob is dual licensed MIT and Apache 2.0. It appears to have a large number of dependencies and good deployment cadence, so not too risky. Approved ✅